app/actions/service_credential_binding_delete.rb

```diff
@@ -21,7 +21,7 @@ def perform_start_delete_actions(binding)
       end
 
       def event_repository
-        Repositories::ServiceBindingEventRepository
+        Repositories::ServiceGenericBindingEventRepository.new('service_binding')
```

Comment:
Everytime this method is called a new `ServiceGenericBindingEventRepository` will be created. It seems it's only being called once in the action, so not a big deal, but may be interesting to cache this one?